src/main/java/frc/robot/RobotContainer.java

```diff
@@ -229,6 +229,9 @@ private void configureButtonBindings() {
             
             new JoystickButton(controlStick, XboxController.Button.kLeftStick.value).whenHeld(slowClimb);
             new JoystickButton(controlStick, XboxController.Button.kRightStick.value).whenHeld(fastClimb);
+
+            Command visionDriverAid = new VisionDriverAid(driverStick, VisionCtrlNetTable.yawToHub);
+            new JoystickButton(driverStick, XboxController.Button.kRightBumper.value).whenHeld(visionDriverAid);
```

Comment:
In David's branch, the driver stick's right bumper is mapped for automatic alignment. Need to figure out a convenient mapping on the driver stick.